spec/draupnir/health_check_spec.rb

```diff
@@ -9,5 +9,20 @@
     expect(JSON.parse(response.body)).to eq("status" => "ok")
     expect(response.headers[:content_type]).to eq("application/json")
     expect(response.headers[:draupnir_version]).to eq(Draupnir::VERSION)
+
+    # Verify that we have a well-formed version, ie. valid semver value
+    expect(response.headers[:draupnir_version]).not_to eql("0.0.0")
+    expect(response.headers[:draupnir_version]).to match(/^\d+\.\d+\.\d+$/)
+  end
+
+  context "with old client version" do
+    it "responds with 'OK'" do
+      begin
+        client.request(:get, "/images", nil, draupnir_version: "0.0.0")
```

Comment:
oh yeah sorry, the healthcheck endpoint doesn't require the version header to be set. confusing 😬